src/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+.important,
+.interesting,
+.surprising,
+#header,
+#footer {
+  width: 300px;
+  height: 100px;
+  margin: auto;
+}
+
+.important:hover,
+.interesting:hover,
+.surprising:hover,
+#header:hover,
+#footer:hover {
+  width: 200px;
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/75342536-49d59580-589f-11ea-88ce-45220fd137d6.png)

Reply:
Changed width to height